docs/modules/con-community-filters.adoc

```diff
@@ -0,0 +1,10 @@
+// file included in the following:
+//
+// index.adoc
+
+[id='con-community-filters-{context}']
+= Community filters
+
+[role="_abstract"]
+Community contributed filters are showcased in the
```

Comment:
Should we be saying something about support levels here? I.e. use at your own risk...